app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  USERNAME_FORMAT = /\A[a-zA-Z0-9]+\z/.freeze
+  has_secure_password
+  validates :password, presence: true
+  validates :username,
+            presence: true,
+            uniqueness: { case_sensitive: true },
+            length: { minimum: 5, maximum: 15 },
```

Comment:
none of this is working - I was able to set a 3 char password

Reply:
Oh, I did not add validation for password length. The username validations are working correct?